src/export/pdf/page.rs

```diff
@@ -267,7 +276,35 @@ impl PageContext<'_, '_> {
                 }
             }
 
-            self.content.set_line_width(stroke.thickness.to_f32());
+            self.content.set_line_width(thickness.to_f32());
+            if self.state.stroke.as_ref().map(|s| &s.line_cap) != Some(line_cap) {
+                let cap = match line_cap {
```

Comment:
no need for a binding here, the move directly in `set_line_cap`'s args is fine I think